sdk/loadtestservice/azure-developer-loadtesting/swagger/README.md

```diff
@@ -1,49 +1,69 @@
-      where: $["paths"]["/loadtests/{testId}/files"].get
-      transform: $["operationId"] = "Test_ListTestFiles"
-    - from: swagger-document
-      where: $["paths"]["/testruns/sortAndFilter"].get
-      transform: $["operationId"] = "TestRun_ListTestRuns"
-    - from: swagger-document
-      where: $["paths"]["/testruns/{testRunId}"].patch
-      transform: $["operationId"] = "TestRun_CreateOrUpdateTest"
-    - from: swagger-document
-      where: $["paths"]["/serverMetricsConfig/{name}"].get
-      transform: $["operationId"] = "ServerMetrics_GetServerMetricsConfigByName"
-    - from: swagger-document
-      where: $["paths"]["/serverMetricsConfig/{name}"].delete
-      transform: $["operationId"] = "ServerMetrics_DeleteServerMetricsConfig"
+      where: $["paths"]["/tests/{testId}"].patch
+      transform: $["operationId"] = "Test_CreateOrUpdateTest";
+    - from: swagger-document 
+      where: $["paths"]["/tests/{testId}"].delete
+      transform: $["operationId"] = "Test_DeleteTest";
+    - from: swagger-document 
+      where: $["paths"]["/tests/{testId}"].get
+      transform: $["operationId"] = "Test_GetTest";
+    - from: swagger-document 
+      where: $["paths"]["/tests"].get
+      transform: $["operationId"] = "Test_ListTests";
+    - from: swagger-document 
+      where: $["paths"]["/test-runs/{testRunId}/metric-dimension/{name}/values"].get
+      transform: $["operationId"] = "TestRun_ListMetricDimensionValues";
+    - from: swagger-document 
+      where: $["paths"]["/tests/{testId}/files/{fileName}"].put
+      transform: $["operationId"] = "Test_UploadTestFile";
+    - from: swagger-document 
+      where: $["paths"]["/tests/{testId}/files/{fileName}"].get
+      transform: $["operationId"] = "Test_GetTestFile";
+    - from: swagger-document 
+      where: $["paths"]["/tests/{testId}/files/{fileName}"].delete
+      transform: $["operationId"] = "Test_DeleteTestFile";
+    - from: swagger-document 
+      where: $["paths"]["/tests/{testId}/files"].get
+      transform: $["operationId"] = "Test_ListTestFiles";
+    - from: swagger-document 
+      where: $["paths"]["/test-runs/{testRunId}"].delete
+      transform: $["operationId"] = "TestRun_DeleteTestRun";
+    - from: swagger-document 
+      where: $["paths"]["/test-runs/{testRunId}"].patch
+      transform: $["operationId"] = "TestRun_CreateOrUpdateTestRun";
+    - from: swagger-document 
+      where: $["paths"]["/test-runs/{testRunId}"].get
+      transform: $["operationId"] = "TestRun_GetTestRun";
+    - from: swagger-document 
+      where: $["paths"]["/test-runs/{testRunId}/files/{fileName}"].get
+      transform: $["operationId"] = "TestRun_GetTestRunFile";
+    - from: swagger-document 
+      where: $["paths"]["/test-runs"].get
+      transform: $["operationId"] = "TestRun_ListTestRuns";
+    - from: swagger-document 
+      where: $["paths"]["/test-runs/{testRunId}:stop"].post
+      transform: $["operationId"] = "TestRun_StopTestRun";
```

Comment:
Why are we doing all these transforms to the operationIds? Can't we use the operationIds as is?

Reply:
Agree with mike. The more directives you use in README.md, the harder you maintain it in the future